pallets/anchor/src/test_utils.rs

```diff
@@ -32,41 +29,29 @@ pub fn setup_zk_circuit(
 	relayer_bytes: Vec<u8>,
 	commitment_bytes: Vec<u8>,
 	pk_bytes: Vec<u8>,
-	src_chain_id: u32,
-	fee_value: u32,
-	refund_value: u32,
+	src_chain_id: u128,
```

Comment:
Why make this u128 and not u32?

Reply:
Hmm, I think we were unsure about this type. In the note, its a hex string: https://github.com/webb-tools/webb.js/blob/master/packages/wasm-utils/src/note/mod.rs#L115, so I assumed it could be uint256 on ethereum. But I see maybe thats not good thinking. Do you propose fixing it to u32 on every chain, as well as in the note and other places?

Reply:
Yea we should discuss, likely we should fix to u32 and update everywhere for the next round of deployment/testing.

Reply:
Though, I have updated the chain identifier structure in resource IDs to consist of 6 bytes of data so we may want to consider adopting a more complex scheme here for the future. Namely, we currently are reserving 4 bytes for the chain ID (u32) and 2 bytes for the chain type (u16).

Reply:
For example, since the chain ID is taken to be a public input. We could hash the `(chain_id_type, chain_id) % u32::max()` or `FIELD_SIZE` and use this as the value meant to be committed to inside deposit notes.

Reply:
The goal with the chain ID type is to add separation between different chains with the same IDs such as parachains with ID 7 and EVM with ID 7 and Cosmos with ID 7.

Reply:
I moved this to an issue: https://github.com/webb-tools/arkworks-gadgets/issues/149. We can continue discussion there